src/Tickets/Seating/app/blockEditor/hook-callbacks.js

```diff
@@ -63,6 +63,10 @@ export const filterHeaderDetails = (items, clientId) => {
  * @return {Object} The body of the request with the seating details.
  */
 export const filterSetBodyDetails = (body, clientId) => {
+	// On first save of a ticket, lock the Layout.
+	// Doesn't matter if ASC or GAC, they layout should be locked.
```

Comment:
You can either short this comment into one line and use `//` syntax, or keep it multi line and use the `/*` syntax.
Multi-line comments should not use `//`, but `/*`.